src/docs/asciidoc/common/cloud-services/standard-user-workflow/quick-start.adoc

```diff
@@ -1,2 +1,192 @@
+
+Use the Standard User Workflow can be done in 3 parts :
+
+* Use the Standard User Workflow in your worker
+* Add properties to configure the Standard User Workflow
+* Create the client part
+
+== Bootstrap
+
+In this Quickstart, we will begin with a generated application using the CLI. So you can run :
+
+[source, console]
+----
+$ npm init @zetapush myApp
+----
+
+We also use the package `@zetapush/user-management` so you can install it with :
+
+[source, console]
+----
+$ npm install --save @zetapush/user-management
+----
+
+== Worker part
+
+Your worker contains just this code :
+
+.worker/index.ts
+[source,typescript,linenums]
+----
+import { Module } from '@zetapush/core';
+import { StandardUserManagementModule, StandardUserWorkflow, ConfirmationUrlHttpHandler } from '@zetapush/user-management';
+
+@Module({
+  imports: [StandardUserManagementModule], <1>
+  expose: {
+    user: StandardUserWorkflow, <2>
+    http: ConfirmationUrlHttpHandler <3>
+  }
+})
+export default class Api {}
+----
+<1> First you need to export a class (named _Api_ here) that is a module (configured with the _@Module()_ annotation). You import the **StandardUserManagementModule** to use it into your application.
+<2> To call, from the front side, the Standard User Workflow cloud functions, you need to expose it (here in the namespace named _user_).
+<3> We also expose _ConfirmationUrlHttpHandler_ to instanciate a web server in your worker that will listen the account confirmation event (When the user clicks on the link in the email).
+
+That all for the worker !
+
+== Properties
+
+The **Standard User Workflow** needs to be configured. So create a `application.json` file at the root of your application :
+
+.application.json
+[source,json,linenums]
+----
+{
+  "email": {
+    "from": "celtia@zetapush.com" <1>
+  },
+  "registration": {
+    "confirmation": {
+      "base-url": "http://localhost:2999", <2>
+      "email": {
+        "subject": "Please confirm your account registration"
+      },
+      "success-url": "http://localhost:3000#login", <3>
+      "failure-url": "http://localhost:3000#confirmation-failed" <4>
+    }
+  },
+  "mailjet": {  <5>
+    "apikey-public": "", 
+    "apikey-private": ""
+  }
+}
+----
+<1> We configure the email sender
+<2> the `base-url` property is necessary to run the application in local. This property will soon be optional.
+<3> Link of your application where the user will be redirected when he confirms his account.
+<4> Link of your application where the user will be redirected when the confirmation failed.
+<5> Your Mailjet properties.
+
+
+== Client part
+
+Finally, we just need to call the cloud functions to create a user account, log in and log out from the client part.
+You can replace the `front/index.js` file by the following content :
```

Comment:
You should explain that in this sample, the user that will be created is hard-coded in the javascript. The aim is not to develop a fully-functional front-end but to understand front -> worker interaction.

Or maybe the sample should use a more complete code with also a complete UI like in the samples (login form + signup form).